run_dir/static/js/project_samples.js

```diff
@@ -354,6 +354,13 @@ function load_all_udfs(){
         safeobj(key).html(auto_format(value));
       }
 
+      // highlight if project is shared
+      else if (prettify(key) == 'shared') {
+        if (value != 'No') {
+          $('#type').append('<span class="label label-warning">Shared</span>');
```

Comment:
if I make a left margin, then the first label will also have a space in front of it but we don't want it. The right margin will only make visible space for the second label